brave/browser-laptop
# Download Item Removal Confirmation

## Test Plan
- Download a file
- try to delete it from the `downloadsBar`
- notice new confirmation bar
  - click `ok` to remove the file
  - move the mouse outside the `downloadItem` panel to hide the confirmation bar.

## Automated tests
`npm run test -- --grep="downloadItem"`

## Description
Fixes #2604


How it works:
![download-remove-confirmation](https://cloud.githubusercontent.com/assets/347657/22621031/3cd2fad0-eb1a-11e6-8e14-63f9c477e566.gif)
